### PR TITLE
Wolfe site refresh

### PR DESCRIPTION
**Pull request by alexwolfe**
@naomiblack @ericjim @chalin 

Ok, here's my branch. It's not ready for merging yet, I'm still adding some stuff but creating a PR so you an get ahead of reviewing the changes. Will be adding more items tomorrow morning... 👍

### CODE REVIEW

File: public/resources/css/base/_colors.scss [hunk 1]
```diff
@@ -0,0 +1,279 @@
+/*
```
Comment (ericjim) [inline public/resources/css/base/_colors.scss]: Thank goodness.

**Comment by alexwolfe** [comment]
Merging with master broke travis :(  
Hope someone is fixing this issue?

**Comment by Foxandxss** [comment]
On my list for today's fixes.

**Comment by alexwolfe** [comment]
@Foxandxss awesome, thanks 👍

**Comment by naomiblack** [comment]
merged

**Pull request closed without merging**